ui/raidboss/data/06-ew/dungeon/stigma_dreamscape.ts

```diff
@@ -0,0 +1,257 @@
+import outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  lastBoss: boolean;
+}
+
+const limitCutNumberMap: { [id: string]: number } = {
+  '004F': 1,
+  '0050': 2,
+  '0051': 3,
+  '0052': 4,
+};
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheStigmaDreamscape,
+  timelineFile: 'stigma_dreamscape.txt',
+  initData: () => {
+    return {
+      lastBoss: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'Dreamscape Side Cannons Left',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6320', source: 'Proto-Omega', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'Dreamscape Side Cannons Right',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6321', source: 'Proto-Omega', capture: false }),
+      response: Responses.goRight(),
+    },
+    {
+      id: 'Dreamscape Forward Interceptors',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6322', source: 'Proto-Omega', capture: false }),
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Dreamscape Rear Interceptors',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6324', source: 'Proto-Omega', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go front of boss',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Chemical Missile',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '008B' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Dreamscape Electric Slide',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0121' }),
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.target!();
+        return output.allies!({ target: matches.target });
+      },
+      outputStrings: {
+        target: {
+          en: 'Stack + Knockback on YOU!',
+        },
+        allies: {
+          en: 'Stack + knockback on ${target}',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Guided Missile',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '0011' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Guided Missile on YOU',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Mustard Bomb',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '632B', source: 'Proto-Omega' }),
+      response: Responses.tankBuster(),
```

Comment:
I haven't tested whether any of the busters in here splash, since I've only run this with randoms.